src/trace.h

```diff
@@ -48,6 +48,9 @@
 #if defined(HAVE_FSEEKO64) && defined(WIN32)
 #  define PRINTF_CAST_U64(x) ((off64_t) (x))
 #  define PRINTF_FORMAT_U64 "%I64u"
+#elif defined(HAVE_FSEEKI64) && defined(WIN32)
+#  define PRINTF_CAST_U64(x) ((__int64) (x))
```

Comment:
Mingw (32-bit) on my system has fseeko64 but not _fseeki:

-- Looking for fseeko
-- Looking for fseeko - not found
-- Looking for fseeko64
-- Looking for fseeko64 - found
-- Looking for _fseeki64
-- Looking for _fseeki64 - not found